pkg/server/api/enroll.go

```diff
@@ -151,6 +151,14 @@ func (c *Api) sign(keyId string, principals []string, serial uint64, certType ui
 		Permissions:     getPermissionsForCertType(&c.conf.SSH, certType),
 	}
 
+	if c.conf.Github.Enabled {
```

Comment:
will this work as expected if `github:` section in the config YAML is missing?

Reply:
it should work, as the tests would fail if it didn't